ssl/ssl_lib.c

```diff
@@ -4132,6 +4132,26 @@ EVP_PKEY *SSL_CTX_get0_privatekey(const SSL_CTX *ctx)
         return NULL;
 }
 
+int SSL_CTX_remove_cert_and_key(SSL_CTX *ctx)
+{
+    if(!ctx) {
+        return 0;
+    }
+
+    if(!CRYPTO_THREAD_write_lock(ctx->cert->lock)) {
```

Comment:
Style: space between the `if` and the `(`.

Drop `{` and `}` around single statement bodies like this one.